apps/dsaparam.c

```diff
@@ -128,6 +128,12 @@ int dsaparam_main(int argc, char **argv)
         goto end;
 
     if (numbits > 0) {
+        if (numbits > OPENSSL_DSA_MAX_MODULUS_BITS)
+            BIO_printf(bio_err,
+                       "# Warning: It is not recommended to use more than %d bit for DSA keys.      #\n"
+                       "#          Larger key size may behave not as expected. Your key size is %d! #\n",
+                       OPENSSL_DSA_MAX_MODULUS_BITS, numbits);
```

Comment:
OPENSSL_RSA_MAX_MODULUS_BITS and OPENSSL_DSA_MAX_MODULUS_BITS are 5 digits numbers.
As the warning is only shown on larger values than the limits, the next digit will added at 100.000 bit.
So as long the terminal is using a monospaced font and has at least 80 chars per line it would work.

Removing all hash signs or only the trailing one are also ok.
The previous hash signs would you keep?

Reply:
See 5ea0b76